### PR TITLE
todos in the it specs

### PR DESCRIPTION
**Pull request by liampan**
Mostly just moving stuff around and adding a spec for the shuttering of endpoints

### CODE REVIEW

File: app/errors/ErrorHandler.scala [hunk 1]
```diff
@@ -53,7 +54,10 @@ class ErrorHandler @Inject()(
 
   override def onClientError(request: RequestHeader, statusCode: Int, message: String): Future[Result] = {
     auditClientError(request, statusCode, message)
-    super.onClientError(request, statusCode, message)
+    if (appConfig.shuttered) {
+      implicit val r: Request[String] = Request(request, "")
+      Future.successful(ServiceUnavailable(shutteringPage()))
+    } else super.onClientError(request, statusCode, message)
```
Comment (burns238) [inline app/errors/ErrorHandler.scala:60]: Could be pattern matching

File: app/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -25,10 +25,11 @@
     hmrcHead: HmrcHead,
     hmrcHeader: HmrcInternalHeader,
     hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
-    hmrcScripts: HmrcScripts
+    hmrcScripts: HmrcScripts,
+    appConfig: AppConfig
```
Comment (burns238) [inline app/views/govuk_wrapper.scala.html:29]: Does this mean we can remove it from every view?

**Review by burns238 — APPROVED** [review]

File: app/views/govuk_wrapper.scala.html (hunk 2 above)
Comment (liampan) [reply app/views/govuk_wrapper.scala.html:29]: yes 😍

File: app/errors/ErrorHandler.scala (hunk 1 above)
Comment (liampan) [reply app/errors/ErrorHandler.scala:60]: you could be pattern matching.

**Pull request merged**